lib/builder.js

```diff
@@ -273,6 +273,7 @@ function executeConfigFile(saveForReset, ignoreBaseURL, source) {
   var context = Object.create(global);
   context.System = configLoader;
   context.global = context.GLOBAL = context.root = context;
+  context.require = require;
```

Comment:
Yes, I did that. However node would not search in node_modules, unless m.path is assigned. I had to call Module._nodeModulePaths() - just copied two lines from module.load() code. (the second one assigns m.filename just in case).